### PR TITLE
New blog post on faf-2.2.0

### PR DESCRIPTION
**Pull request by michalfabik**
See commit.

Signed-off-by: Michal Fabik <mfabik@redhat.com>

### CODE REVIEW

**Comment by michalfabik** [comment]
Oh, I probably shouldn't have mentioned the FAF vs. ABRT Analytics thing, considering https://github.com/abrt/abrt.github.io/pull/24.

**Comment by mkutlak** [comment]
#deadlock

File: _posts/2020-03-11-faf-2-2-0.markdown [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+author: Michal Fabík
+visible: 1
+categories: release features faf "ABRT Analytics"
+---
+
+A new version of FAF has just been released. From the user's point of view, you will probably find the following changes the most noticable:
+
++ A markedly sped up `create-problems` action.
++ A more capable CLI with autocompletion and wildcard support
++ Many web UI improvements. <sub>(Still evaluating some changes to the table layout though.)</sub>
```
Comment (mkutlak) [inline _posts/2020-03-11-faf-2-2-0.markdown]: I would drop the `Still evaluating some changes to the table layout though.` part.

File: _posts/2020-03-11-faf-2-2-0.markdown [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
++ The web UI action scheduler is now working.
```
Comment (mkutlak) [inline _posts/2020-03-11-faf-2-2-0.markdown]: `Fixed web UI action scheduler.` instead?

File: _posts/2020-03-11-faf-2-2-0.markdown [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+A more complete change log can be found [here](https://github.com/abrt/faf/blob/2.2.0/CHANGELOG.md).
+
+## Where to get the new release:
+If you want to install the new version of ABRT Analytics, you can get it from our **COPR**: [@abrt/faf-el8](https://copr.fedorainfracloud.org/coprs/g/abrt/faf-el8/)
```
Comment (mkutlak) [inline _posts/2020-03-11-faf-2-2-0.markdown]: `ABRT Analytics` :eyes:

Comment (michalfabik) [reply _posts/2020-03-11-faf-2-2-0.markdown]: Fixed.

File: _posts/2020-03-11-faf-2-2-0.markdown (hunk 2 above)
Comment (michalfabik) [reply _posts/2020-03-11-faf-2-2-0.markdown]: Changed.

File: _posts/2020-03-11-faf-2-2-0.markdown (hunk 1 above)
Comment (michalfabik) [reply _posts/2020-03-11-faf-2-2-0.markdown]: Dropped.

**Comment by mkutlak** [comment]
I will merge this PR if no one has anything against it at 14:00.

**Comment by ernestask** [comment]
What time zone?

File: _posts/2020-03-11-faf-2-2-0.markdown [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+visible: 1
+categories: release features faf
+---
+
+A new version of FAF has just been released. From the user's point of view, you will probably find the following changes the most noticable:
+
++ A markedly sped up `create-problems` action.
++ A more capable CLI with autocompletion and wildcard support
++ Many web UI improvements.
++ Fixed web UI action scheduler.
```
Comment (ernestask) [inline _posts/2020-03-11-faf-2-2-0.markdown]: The punctuation is a bit inconsistent here.

File: _posts/2020-03-11-faf-2-2-0.markdown [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+date:   2020-03-11 14:00:00 +0100
+author: Michal Fabík
+visible: 1
+categories: release features faf
+---
+
+A new version of FAF has just been released. From the user's point of view, you will probably find the following changes the most noticable:
+
++ A markedly sped up `create-problems` action.
++ A more capable CLI with autocompletion and wildcard support
```
Comment (ernestask) [inline _posts/2020-03-11-faf-2-2-0.markdown:13]:
```suggestion
+ A more capable CLI with autocompletion and wildcard support.
```

**Comment by ernestask** [comment]
I have no complaints otherwise.

File: _posts/2020-03-11-faf-2-2-0.markdown (hunk 5 above)
Comment (michalfabik) [reply _posts/2020-03-11-faf-2-2-0.markdown:13]: Went the other way round, as those aren't complete sentences.

**Comment by mgrabovsky** [comment]
Looks good to me as well.

**Comment by mkutlak** [comment]
Oops, I missed the deadline. 

LGTM

Merging in 3, 2, 1...

**Pull request merged**